bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CTabItem.java

```diff
@@ -276,6 +277,20 @@ public boolean getShowClose() {
 	checkWidget();
 	return showClose;
 }
+
+/**
+ * Returns <code>true</code> to indicate that the dirty indicator should be shown.
+ * Otherwise return <code>false</code>.
+ *
+ * @return <code>true</code> if the dirty indicatorn should be shown
+ *
+ * @since 3.129
+ */
+public boolean getShowDirty() {
+	checkWidget();
+	return showClose;
+}
+
```

Comment:
@schneidermic0 I know that you said that there is still a lot to improve but I think one of these may be a copy-paste bug so I'll bring it up (and more in the same method):

- Why `return showClose`? (_Copy-paste_ bug?)
- I'd call it `is...` and not `get...` (I don't know why `getShowClose()` uses `get...`). 
- There is a typo in the javadoc:

```suggestion
/**
 * Returns <code>true</code> to indicate that the dirty indicator should be shown.
 * Otherwise return <code>false</code>.
 *
 * @return <code>true</code> if the dirty indicator should be shown
 *
 * @since 3.129
 */
public boolean isShowDirty() {
	checkWidget();
	return showDirty;
}

```

Reply:
+1 for changing it to `isShowDirty()`. I like that convention better than using `get...` for `boolean`s.

Reply:
I have renamed it to `isDirtyShown()` with https://github.com/eclipse-platform/eclipse.platform.swt/pull/1632/commits/8f9835592ac738b47ed51eff8cebc9d2dce467b9